backend/src/services/tasks.service.ts

```diff
@@ -17,6 +17,10 @@ export class TasksService {
     await this.tasksRepository.insert(task);
   }
 
+  async updateTask(taskId: number, task: Task): Promise<void> {
+    await this.tasksRepository.update({ id: taskId }, task);
```

Comment:
Can we surround the calls to the repository with a `try... catch` block and log exceptions for better observability?